connect/runtime/src/test/java/org/apache/kafka/connect/storage/KafkaStatusBackingStoreFormatTest.java

```diff
@@ -216,7 +216,7 @@ public void putTopicStateRetriableFailure() {
         }).when(kafkaBasedLog).send(eq(key), valueCaptor.capture(), any(Callback.class));
 
         store.put(topicStatus);
-        verify(kafkaBasedLog, times(2)).send(any(), any(), any());
+        verify(kafkaBasedLog, timeout(1000).times(2)).send(any(), any(), any());
```

Comment:
Hi @vamossagar12 thanks for the PR.
Just for my own understanding, how was the timeout value chosen?

Reply:
Are we concerned this could still be flaky?

Reply:
@machi1990 it was originally 300 ms but @viktorsomogyi suggested to bump it up to 1s https://github.com/apache/kafka/pull/13594#discussion_r1175222973. @jolshan i haven't seen this test fail in the last few builds with this PR and https://github.com/apache/kafka/pull/13594.

Reply:
thanks @vamossagar12 for the reply. 1s seems like a high enough value timeout so it might be enough to make this test resilient.